scripts/test_env.py

```diff
@@ -0,0 +1,191 @@
+#!/usr/bin/env python3
+"""
+Setup or tear down a bare metal
+environment running a debug build of Iroha.
+"""
+
+import argparse
+import ipaddress
+import json
+import logging
+import os
+import pathlib
+import shutil
+import subprocess
+import sys
+
+
+class Network:
+    """
+    A network of bootstrapped peers to run on bare metal.
+    """
+    def __init__(self, setup_dir: pathlib.Path, host_addr: ipaddress.IPv4Address, n_peers: int = 4):
+        logging.info("Setting up test environment...")
+        self.setup_dir = setup_dir
+        self.n_peers = n_peers
+        peers_dir = setup_dir.joinpath("peers")
+        os.mkdir(peers_dir)
+        try:
+            shutil.copy2("./configs/peer/config.json", peers_dir)
+            shutil.copy2("./configs/peer/genesis.json", peers_dir)
+            shutil.copy2("./configs/peer/validator.wasm", peers_dir)
+        except FileNotFoundError:
+            logging.error("Some of the config files are missing. \
+                          Please provide them in the `./configs/peer` directory")
+            sys.exit(1)
+        try:
+            shutil.copy2("./target/debug/iroha", peers_dir)
+        except FileNotFoundError:
+            logging.error("Please build the `iroha` by running `cargo build --bin iroha`")
+            sys.exit(1)
+
+        self.peers = [Peer(setup_dir, host_addr, i) for i in range(n_peers)]
+
+        os.environ["IROHA2_CONFIG_PATH"] = str(setup_dir.joinpath("peers/config.json"))
+        os.environ["IROHA2_GENESIS_PATH"] = str(setup_dir.joinpath("peers/genesis.json"))
+        os.environ["IROHA_GENESIS_ACCOUNT_PUBLIC_KEY"] = self.peers[0].public_key
+        os.environ["IROHA_GENESIS_ACCOUNT_PRIVATE_KEY"] = self.peers[0].private_key
+
+        self.generate_trusted_peers()
+    
+    def generate_trusted_peers(self):
```

Comment:
I wouldn't mind if this function were inlined. A linear sequence of steps is often easier to follow than a set of functions which are used and called once.